examples/Demo/FluentUI.Demo.Client/Documentation/Components/Spacing/Spacing.md

```diff
@@ -0,0 +1,142 @@
+
+Spacing utilities that apply to all breakpoints, from `xs` to `xl`,
+have no breakpoint abbreviation in them. 
+The classes are named using the format `{property}{direction}-{size}`
+for `xs` and `{property}{direction}-{breakpoint}-{size}` for `sm`, `md`, `lg`, and `xl`.
+
+The **properties**:
+
+- `m` for classes that set `Margin`.
+- `p` for classes that set `padding`.
+
+The **direction** property applies to:
+
+- `t` for `margin-top` or `padding-top`.
+- `b` for `margin-bottom` or padding-bottom`.
+- `l` for `margin-left` or padding-left`.
+- `r` for `margin-right` or padding-right`.
+- `s` for `margin-left`/`padding-left` (in LTR mode) and `margin-right`/`padding-right` (in RTL mode).
+- `e` for `margin-right`/`padding-right` (in LTR mode) and `margin-left`/`padding-left` (in RTL mode).
+- `x` for `margin-left`/`padding-left` and `margin-right`/`padding-right`.
+- `y` for `margin-top`/`padding-top` and `margin-bottom`/`padding-bottom`.
+- `a` for all 4 sides.
+
+## Size
+
+The size changes with an interval of, by default, **4 pixels** (see the CSS variables below).
+
+**Positive**
+
+- `0` sets `margin` or `padding` to `0`
+- `1` sets `margin` or `padding` to `4px`
+- `2` sets `margin` or `padding` to `8px`
+- `3` sets `margin` or `padding` to `12px`
+- `4` sets `margin` or `padding` to `16px`
+- `5` sets `margin` or `padding` to `20px`
+- `6` sets `margin` or `padding` to `24px`
+- `7` sets `margin` or `padding` to `28px`
+- `8` sets `margin` or `padding` to `32px`
+
+**Negative**
+
+- `n1` sets `margin` or `padding` to `-4px`
+- `n2` sets `margin` or `padding` to `-8px`
+- `n3` sets `margin` or `padding` to `-12px`
+- `n4` sets `margin` or `padding` to `-16px`
+- `n5` sets `margin` or `padding` to `-20px`
+- `n6` sets `margin` or `padding` to `-24px`
+- `n7` sets `margin` or `padding` to `-28px`
+- `n8` sets `margin` or `padding` to `-32px`
+
+## Example
+
+```css
+/* margin-top: 0; */
+.mt-0 {
+  margin-top: var(--spacingVerticalNone) !important; 
+}
+
+/* margin-left: 4px; */
+.ml-1 {
+  margin-left: var(--spacingHorizontalXS) !important;
+}
+
+/* margin-right: 8px; margin-left: 8px; */
+.px-2 {
+  padding-right: var(--spacingHorizontalS) !important;
+  padding-left:  var(--spacingHorizontalS) !important;
+}
+
+/* padding: 12px 12px; */
+.pa-3 {
+  padding: var(--spacingVerticalM) var(--spacingHorizontalM) !important;
+}
+```
+
+## `Margin` and `Padding` component attributes
+
+All **FluentUI Blazor components** contain the `Margin` and `Padding` attributes.
+
+You can specify a **CSS value** respecting the CSS [padding](https://developer.mozilla.org/docs/Web/CSS/padding)
+or CSS [margin](https://developer.mozilla.org/docs/Web/CSS/margin) pattern.
+Or you can specify a **class name** like defined above on this page.
+
+> If you use one of these properties to include a complete style (e.g. `style=“margin: 10px;”`)
+> then this style will be ignored. The property `Style` can be used to include a complete style.
```

Comment:
This is not really clear to me. Needs to be reworded. Maybe show an examples of what would be ignored and when